test_data/rules/require_trailing_commas.dart

```diff
@@ -39,7 +39,7 @@ class RequireTrailingCommasExample {
       : assert(true,
             'A very very very very very very very very long string'); // LINT
 
-  void method1(Object param1, Object param2, {Object? param3, Object? param4}) {}
+  void method1(Object p1, Object p2, {Object? param3, Object? param4}) {}
```

Comment:
Curious to know why you changed the names of half the parameters (rather than either none or all). It doesn't change the semantics of the test, just seems inconsistent.

Reply:
Before the change the line length was 81 and the formatting (assumed by this rule) splitted the line.